__tests__/app.test.js

```diff
@@ -27,75 +27,90 @@ describe("GET /api", () => {
 });
 
-
-
 describe("GET /api/articles/:article_id", () => {
   test("gives status of 200 and responds with the appropriate article", () => {
     return request(app)
       .get("/api/articles/2")
       .expect(200)
       .then(({ body: { chosenArticle } }) => {
         expect(Object.keys(chosenArticle)).toHaveLength(8);
-        expect(chosenArticle).toMatchObject(
-          { 
-            article_id: expect.any(Number),
-            title: expect.any(String),
-            topic: expect.any(String),
-            author: expect.any(String),
-            body: expect.any(String),
-            created_at: expect.any(String),
-            votes: expect.any(Number),
-            article_img_url: expect.any(String),
-          }
-        )
-        
+        expect(chosenArticle).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
       });
   });
 });
 
-
 describe("GET /api/articles", () => {
   test("gives status of 200 and responds with all articles", () => {
     return request(app)
       .get("/api/articles")
       .expect(200)
       .then(({ body: { allArticles } }) => {
         allArticles.forEach((article) => {
-					expect(Object.keys(article)).toHaveLength(8);
-					expect(article).toMatchObject({
-						article_id: expect.any(Number),
+          expect(Object.keys(article)).toHaveLength(8);
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
             title: expect.any(String),
             topic: expect.any(String),
             author: expect.any(String),
             created_at: expect.any(String),
             votes: expect.any(Number),
             article_img_url: expect.any(String),
             comment_count: expect.any(Number),
-					});
-				});
-        
-        
-        expect(allArticles).toBeSortedBy('created_at', { descending: true });
+          });
+        });
+
+        expect(allArticles).toBeSortedBy("created_at", { descending: true });
       });
   });
 });
 
+describe("/api/articles/:article_id/comments", () => {
+  test("Responds with an array of comments for the given article_id", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body: { allArticleComments } }) => {
+        console.log(allArticleComments)
+        allArticleComments.forEach((comment) => {
+          expect(Object.keys(comment)).toHaveLength(6);
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
again, are we expecting any `article_id` or are we expecting these comments to belong to a specific article?